cuml/pca_demo.ipynb

```diff
@@ -15,29 +15,11 @@
    "source": [
```

Comment:
Please clear the output obtained after running the cells in the notebook

Reply:
Done!

Fixed in local.